pages/treeherder.py

```diff
@@ -133,9 +175,10 @@ def clear_pinboard(self):
 
         class Job(PageRegion):
 
-            @property
-            def is_selected(self):
-                return 'selected-job' in self._root.get_attribute('class')
+            def click(self):
+                self._root.click()
+                Wait(self.selenium, self.timeout).until(
+                    lambda _: self.page.job_details.job_result_status)
```

Comment:
I'm not sure I understand your comment- this came in after rebasing #42 and matches production.